plugins/vpp/data_change.go

```diff
@@ -395,51 +395,63 @@ func (plugin *Plugin) dataChangeIface(diff bool, value *interfaces.Interfaces_In
 	changeType datasync.Op) error {
 	plugin.Log.Debug("dataChangeIface ", diff, " ", changeType, " ", value, " ", prevValue)
 
+	var err error
 	if datasync.Delete == changeType {
-		return plugin.ifConfigurator.DeleteVPPInterface(prevValue)
+		err = plugin.ifConfigurator.DeleteVPPInterface(prevValue)
 	} else if diff {
-		return plugin.ifConfigurator.ModifyVPPInterface(value, prevValue)
+		err = plugin.ifConfigurator.ModifyVPPInterface(value, prevValue)
 	}
-	return plugin.ifConfigurator.ConfigureVPPInterface(value)
+	err = plugin.ifConfigurator.ConfigureVPPInterface(value)
```

Comment:
This will also run for delete and modify, basically always. It should be in another `else` case.